install/common-pre-install/hpcm_switch_conf_spine_002_starlord.md

```diff
@@ -0,0 +1,192 @@
+# Copy the text from here
```

Comment:
The title of this file should be more descriptive about the configuration included in the code block below.

The internal system name "starlord" should be removed from the file name (and updated in links).